wordpress-mobile/WordPress-iOS
# Issues/reader search analytics

This PR updates the analytics related to the reader search feature. 

To test:
Set break points where the analytics are bumped.  Confirm that the calls are correctly made and there are no warnings/crashes. 

Needs review: @sendhil would you mind taking this one?
